### PR TITLE
ssh package version change

### PR DESCRIPTION
**Pull request by setaskin**
ssh package version change

### CODE REVIEW

**Comment by jfullerton44** [comment]
should we also update ts to this version?

**Comment by setaskin** [comment]
> should we also update ts to this version?

Oh I didn't know that , I will add and I saw your Pr for higher version. I will pull your changes once yours completed.

**Review by jfullerton44 — COMMENTED** [review]
Can you update https://github.com/microsoft/dev-tunnels/blob/main/ts/package.json#L25 and https://github.com/microsoft/dev-tunnels/blob/main/ts/package-lock.json

File: ts/package-lock.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
 			"name": "@microsoft/dev-tunnels",
 			"license": "MIT",
 			"dependencies": {
-				"@microsoft/dev-tunnels-ssh": "^3.11.11",
-				"@microsoft/dev-tunnels-ssh-tcp": "^3.11.11",
+				"@microsoft/dev-tunnels-ssh": "^3.11.13",
+				"@microsoft/dev-tunnels-ssh-tcp": "^3.11.13",
```
Comment (jfullerton44) [inline ts/package-lock.json:11]: There are some autogenerated parts of this file as well. Can you run 'npm install' in this folder to regenerate this file

**Review by jfullerton44 — CHANGES_REQUESTED** [review]

**Review by jfullerton44 — APPROVED** [review]

**Pull request merged**